<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form1</class>
 <widget class="QWidget" name="Form1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1923</width>
    <height>1083</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QCalendarWidget" name="calendarWidget1">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>571</width>
     <height>401</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label1ofcal">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>1341</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>27</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(0, 0, 0);
color: white;
border-radius:20px;

QPushButton#button{
	background-color: qlineargradient(spread:pad, x1:0, y1:0., x2:1, y2:0, stop:0 rgba(20, 47,78, 219), stop:1 rgba(85, 98, 112, 226));
	color:rgba(255,255,255,210);
	border-radius:50px;
}
QPushButton#button:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0., x2:1, y2:0, stop:0 rgba(40, 67,98, 219), stop:1 rgba(105, 118, 132, 226));
}
QPushButton#button:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color:rgba(105,118,132,200);
}

QPushButton#button_2{
	background-color: rgba(0,0,0,0);
	color:rgba(85,98,112,255);
}
QPushButton#button:hover_2{
	color:rgba(155,168,182,220);
}
QPushButton#button:pressed_2{
	padding-left:5px;
	padding-top:5px;
	background-color:rgba(115,128,142,255);
}</string>
   </property>
   <property name="text">
    <string>                                      SCHEDULE EVENT</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2ofcal">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>1000</y>
     <width>1931</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">BACKGROUND-COLOR: #26425A;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>190</y>
     <width>531</width>
     <height>401</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Background-color:(0,0,0,100)
color:white;
font: 75 12pt &quot;Comic Sans MS&quot;;</string>
    </property>
    <property name="insertPolicy">
     <enum>QComboBox::NoInsert</enum>
    </property>
    <property name="duplicatesEnabled">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>--Select--</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ignitra</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Iris</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mosaic</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Comic Sans MS&quot;;
</string>
    </property>
    <property name="text">
     <string>Event Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Comic Sans MS&quot;;
</string>
    </property>
    <property name="text">
     <string>Event Venue:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Comic Sans MS&quot;;
</string>
    </property>
    <property name="text">
     <string>Time          </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Comic Sans MS&quot;;
</string>
    </property>
    <property name="text">
     <string>From          :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Comic Sans MS&quot;;
</string>
    </property>
    <property name="text">
     <string>To              :</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="timeEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>250</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Background-color:(0,0,0,100);
color:white;
font: 75 12pt &quot;Comic Sans MS&quot;;</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="timeEdit_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>300</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Background-color:(0,0,0,100);
color:white;
font: 75 12pt &quot;Comic Sans MS&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Venue">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>150</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Background-color:(0,0,0,100);
color:white;
font: 75 12pt &quot;Comic Sans MS&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Comic Sans MS&quot;;
</string>
    </property>
    <property name="text">
     <string>Date:</string>
    </property>
   </widget>
   <widget class="QLabel" name="show_date">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Background-color:(0,0,0,100);
color:white;
font: 75 12pt &quot;Comic Sans MS&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>Select a day:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CreateEvent">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>640</y>
     <width>421</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font:  &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);
BORDER-RADIUS: 10PX;
BORDER-COLOR: #26425A;</string>
   </property>
   <property name="text">
    <string>Create Event</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-90</y>
     <width>1931</width>
     <height>1021</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/prefix/bg_edited.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>:/prefix/bg_edited.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>1080</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>:/prefix/bg_edited.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="back7_butt">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
BORDER-RADIUS:10PX;</string>
   </property>
   <property name="text">
    <string>&lt; BACK</string>
   </property>
  </widget>
  <zorder>label_9</zorder>
  <zorder>label_8</zorder>
  <zorder>calendarWidget1</zorder>
  <zorder>label1ofcal</zorder>
  <zorder>label_2ofcal</zorder>
  <zorder>frame</zorder>
  <zorder>label_7</zorder>
  <zorder>CreateEvent</zorder>
  <zorder>back7_butt</zorder>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
